gold-phone-input.html

```diff
@@ -157,9 +157,27 @@
       }
     },
 
-    observers: [
-      '_computeValue(value)'
-    ],
+    listeners: {
+      'input': '_onInput'
+    },
+
+    ready: function() {
+      this.addEventListener('blur', function() {
+        this._handleAutoValidate();
```

Comment:
Would it be possible to override https://github.com/PolymerElements/iron-behaviors/blob/master/iron-control-state.html#L67 instead of adding another listener for `blur`?